src/build.py

```diff
@@ -576,26 +576,42 @@ def GetRepoInfo():
   return info
 
 
+def Which(name):
+  """Find an executable on the system by name. If not found return ''."""
+  # If we want to run this on Windows, we'll have to be smarter.
```

Comment:
SPIRV-Tools solved it like this: https://github.com/KhronosGroup/SPIRV-Tools/pull/132